addons/crm/tests/test_crm_lead.py

```diff
@@ -33,6 +33,7 @@ def test_crm_lead_contact_fields_mixed(self):
         """ Test mixed configuration from partner: both user input and coming
         from partner, in order to ensure we do not loose information or make
         it incoherent. """
+        lang = self.env['res.lang'].search([('code', '=', 'fr_FR')])
```

Comment:
Note that we could set it i setupclass (cls.lang_ref), so that we may re-use it in other tests.